client/src/components/layouts/layoutComponents/Header.js

```diff
@@ -1,36 +1,78 @@
 import React from 'react';
-import { Navbar, Nav } from 'react-bootstrap';
+import {
+  Navbar, Nav, Dropdown, Image, Row, Col, Container,
+} from 'react-bootstrap';
 import { Link } from 'react-router-dom';
 import { useSelector } from 'react-redux';
-import { BsPeopleCircle } from 'react-icons/bs';
+import { BsFillPersonFill } from 'react-icons/bs';
+import { AiFillSetting, AiOutlineLogout } from 'react-icons/ai';
 import { PropTypes } from 'prop-types';
 
 import './Header.css';
+import { NavbarBrand } from 'reactstrap';
+import { theme } from '../../landing-pages/theme';
 
 /**
  * This function provides header for Layout.js
  * @returns Header component to be rendered in Layout.js
  */
-const Header = ({ data }) => {
-  const userName = useSelector(
-    (state) => `${state.team?.teamName} @ ${state.team?.orgName}`,
-  );
+const Header = ({ data, setLogoutAlert }) => {
+  const { teamName, orgName, profilePic } = useSelector((state) => state.team);
   // TODO: Remove hard-coded team id and publications id from the links
   return (
     <>
       <Navbar className="header" fixed="top">
-        <Navbar.Brand>
+        <NavbarBrand href="/">
           <Link className="header-brand" to={data.dashboardURL}>
-            {data.title}
+            <h2 style={{ color: theme.dark, fontFamily: 'Arial' }}>
+              RE
+              <b style={{ color: theme.primary }}>SEARCH</b>
+              IFY
+            </h2>
           </Link>
-        </Navbar.Brand>
+        </NavbarBrand>
         <Nav className="mr-auto" />
         <Nav>
-          <Link className="header-link" to="/dashboard/profile">
-            <BsPeopleCircle className="header-profile-icon" />
-            {' '}
-            {userName}
-          </Link>
+          <Dropdown drop="down" alignRight="end" className="header-link">
+            <Dropdown.Toggle
+              as={BsFillPersonFill}
+              className="dashboard-dropdown-toggle"
+            />
+            <Dropdown.Menu>
+              <Dropdown.Item className="dashboard-dropdown-login-details">
+                <Container fluid>
+                  <Row>
+
+                    <Image
+                      src={profilePic}
+                      roundedCircle
+                      height="60px"
+                      width="60px"
+                    />
+
+                    <Col>
+                      {teamName}
+                      <br />
+                      {orgName}
+                    </Col>
+                  </Row>
+                </Container>
+
+              </Dropdown.Item>
+              <Dropdown.Divider />
+              <Dropdown.Item eventKey="2" href="/dashboard/profile">
```

Comment:
Can you use useHistory hook to redirect to the setting page? I think it's re-rendering the page now and it sends an API call the fetch the sidebar data

Reply:
hmm might have 

> The logout isn't working properly. I think you forgot to pass the setLogoutAlert function to Header.js ?

Might have messed up something when splitting the pr.